translations/fr.json

```diff
@@ -121,14 +121,14 @@
   "domain.record.status.under_development": "En cours de création",
   "domain.record.updateFrequency.asNeeded": "Lorsque nécessaire",
   "domain.record.updateFrequency.continual": "Continue",
-  "domain.record.updateFrequency.day": "jour",
+  "domain.record.updateFrequency.day": "{count} fois par jour",
   "domain.record.updateFrequency.irregular": "Irrégulière",
-  "domain.record.updateFrequency.month": "mois",
+  "domain.record.updateFrequency.month": "{count} fois par mois",
   "domain.record.updateFrequency.notPlanned": "Non planifiée",
   "domain.record.updateFrequency.periodic": "Récurrente",
   "domain.record.updateFrequency.unknown": "Inconnue",
-  "domain.record.updateFrequency.week": "semaine",
-  "domain.record.updateFrequency.year": "an",
+  "domain.record.updateFrequency.week": "{count} fois par semaine",
+  "domain.record.updateFrequency.year": "{count} fois par an",
```

Comment:
Looking at the (after) screenshot again, I'm telling myself we should probably translate the displayed numbers into words as well. Can we assume from [here](https://github.com/geonetwork/geonetwork-ui/blob/main/libs/api/metadata-converter/src/lib/iso19139/codelists/update-frequency.mapper.ts#L46) that 4 will stay the highest number?

Reply:
I was thinking the same! I would only maybe translate up to 2 or 3. Theoretically it could be 12 times per year but it's fine not to translate the number into a word.

Reply:
So, finally I just put `1` into words, as the `messageformat` only distinguishes singular and plural out of the box.